docker/docker-entrypoint-release.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Dockerfile entrypoint
+set -e
+
+source ~/.bashrc
+
+# Activate conda environment
+source activate garage
```

Comment:
How does this differ from the CI entrypoint?

Reply:
I removed this file and with some light modifications I'm using the CI entry point.